webpack/components/extensions/HostDetails/Tabs/ModuleStreamsTab/ModuleStreamsTab.js

```diff
@@ -324,6 +330,16 @@ export const ModuleStreamsTab = () => {
 
   const activeFilters = [statusSelected, installStatusSelected];
   const defaultFilters = [MODULE_STREAM_STATUS, MODULE_STREAM_INSTALLATION_STATUS];
+
+  const filterFunctions = () => {
+    if (statusSelected !== MODULE_STREAM_STATUS) {
+      setStatusSelected(MODULE_STREAM_STATUS);
+    }
+    if (installStatusSelected !== MODULE_STREAM_INSTALLATION_STATUS) {
+      setInstallStatusSelected(MODULE_STREAM_INSTALLATION_STATUS);
+    }
```

Comment:
I don't think the `if` statements are needed.

```suggestion
    setStatusSelected(MODULE_STREAM_STATUS);
    setInstallStatusSelected(MODULE_STREAM_INSTALLATION_STATUS);
```